packages/speech/smoke-test/speech_smoke_test.js

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017, Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+describe('SpeechSmokeTest', function() {
+
+  it('successfully makes a call to the service', function(done) {
```

Comment:
Ha, so there are two problems. One is that the tests are named the wrong thing, and another is that they don't work. :-)

In this case, let's not block on that (those tests are hot off the presses from an autogen change), so long as we do not run them. We will fix both of those problems at once.